docs/user/integration/integrate-jaeger.md

```diff
@@ -0,0 +1,181 @@
+# Install custom Jaeger in Kyma
```

Comment:
All files and folders in the "integration" folder start with "integrate-*", does that bring value? I would say it is redundant

Reply:
Yes, because it specifies what the document is about. It's not about CLS or Loki in general, but about integrating CLS or Loki.